bucket/vtfedit.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.3.3",
+    "description": "A VMT/VTF viewer, editor, and creation tool.",
+    "homepage": "http://nemesis.thewavelength.net/index.php?c=169",
+    "license": "Freeware,GPL-2.0-only,LGPL-2.1-only",
+    "notes": ".NET Framework 3.5 is required; VTFEdit uses .NET Framework 2.0, which 3.5 also installs.",
+    "url": "https://valvedev.info/tools/vtfedit/vtfedit.zip",
+    "hash": "eb2329d105f57057f6fc6d38c34a741c2abc9f05125e80e8b0f8ab5f269a3c49",
+    "bin": "VTFEdit.exe",
+    "suggest": {
+        "vcredist": "extras/vcredist2005"
+    },

```

Comment:
```suggestion
```

Reply:
Should there be nothing used in Scoop if vcredist2005 is required? I seen "depends", but last I installed vcredist from Scoop, you'd be asked to uninstall after it's done.

Reply:
Never depends on vcredist.